src/smc-util/misc.coffee

```diff
@@ -1458,10 +1458,15 @@ exports.map_diff = (a, b) ->
     return c
 
 # limit the values in a by the values of b
+# or just by b if b is a number
 exports.map_limit = (a, b) ->
     c = {}
-    for k, v of a
-        c[k] = Math.min(v, (b[k] ? Number.MAX_VALUE))
+    if typeof b == 'number'
```

Comment:
Please change to `typeof(b)` -- I don't like to use the no parens coffee-script syntax except for callbacks.